### ISSUE TITLE
hanging pools, heap dump fails

### ISSUE THREAD

**Issue by calestyo** [dCache/dcache#2595]
Hey.

dCache 2.15.8.

FYI
We've had some pools got hanging (see attached logs)... perhaps that helps you. A thread dump should be included.

Also
```
# dcache dump heap pool_lcg-lrz-dc07_0 pool_lcg-lrz-dc07_0.heap
Failed to resolve .

```
fails.

Cheers,
Chris.

[pool_lcg-lrz-dc07_0.log.xz.txt](https://github.com/dCache/dcache/files/335283/pool_lcg-lrz-dc07_0.log.xz.txt)
(This is not a txt but an .xz... github is stupid with its file endings)

**Comment by gbehrmann** [comment]
That's a different issue. You are likely in a directory in which the dCache users doesn't have write permissions.

**Comment by gbehrmann** [comment]
...and by a different issue I was referring to some other thread in which somebody had problems with dumping pools :-) Should have scrolled to the top to check if this was a new issue or not.

**Comment by calestyo** [comment]
> That's a different issue. You are likely in a directory in which the dCache users doesn't have write permissions.

Ah,... yes I wasn't. But it's not clear (neither from the tool nor from the error message, that this is reuquired).

**Comment by gbehrmann** [comment]
Yeah, the original version of the patch would have written "pool_lcg-lrz-dc07_0.heap is not writable to user dcache", but there were concerns over that error message (see https://rb.dcache.org/r/9303/).

**Comment by calestyo** [comment]
Concerns? Now it says "Failed to resolve ." which is completely misleading…
It could simply say "No permissions to write <pathname> as user <java domain user>." and everyone would probably get it.
